### DISCUSSION TITLE
webhooks

### DISCUSSION THREAD

**Discussion by vbakarian** [UltimaHoarder/UltimaScraper#1169] [Q&A]
im trying to add my discord webhook to the config code im just a bit confused as to where i should put it exactly. 

any help would be greatly appreciated

**Comment by FelisFulgens** [comment]
I copied the discord webhook url into:
Settings -> webhooks -> download_webhook -> succeeded -> webhooks: ["your webhook url here"]

Where the red line is.

![webhooks example](https://user-images.githubusercontent.com/83669244/118370189-40038500-b56c-11eb-9364-54d1f0a14591.JPG)

Hope that helps!

**Reply by FelisFulgens** [reply]
That image is compressed as hell but I think you can still figure out what's going on. lmk if you can't.

**Reply by vbakarian** [reply]
Ahhh it needed to go in the succeeded section. Perfect thanks!!
